esmero-cantaloupe/Dockerfile

```diff
@@ -145,56 +144,22 @@ RUN curl -OL https://github.com/jasper-software/jasper/archive/version-2.0.32/ja
   && make test \
   && make install
 
-WORKDIR /tmp
-RUN wget $PKGSOURCE && \
-    lzip -d -c GraphicsMagick-$PKGVER.tar.lz | tar -xvf - && \
-    cd GraphicsMagick-$PKGVER && \
-    ./configure \
-      --build=$CBUILD \
-      --host=$CHOST \
-      --prefix=/usr \
-      --sysconfdir=/etc \
-      --enable-magick-compat \
-      --mandir=/usr/share/man \
-      --infodir=/usr/share/info \
-      --localstatedir=/var \
-      --enable-shared \
-      --disable-static \
-      --with-modules \
-      --with-threads \
-      --with-webp=yes \
-      --with-tiff=yes \
-      --with-jpeg=yes \
-      --with-jp2=yes \
-      --with-png=yes \
-      --with-xml=yes \
-      --with-wmf=yes \
-      --with-ttf \
-      --with-gs-font-dir=/usr/share/fonts/Type1 \
-      --with-quantum-depth=$QUANTUMDEPTH && \
-    make && \
-    make install && \
-    cd /tmp && \
-    rm -rf GraphicsMagick-$PKGVER && \
-    rm GraphicsMagick-$PKGVER.tar.lz
-  
-
 # Cantaloupe
 WORKDIR /tmp
 
 RUN apt-get update && apt-get install -y --no-install-recommends maven unzip
 
-RUN curl -OL https://github.com/cantaloupe-project/cantaloupe/archive/refs/heads/release/4.1.zip \
+RUN curl -OL https://github.com/cantaloupe-project/cantaloupe/archive/refs/heads/release/5.0.zip \
```

Comment:
This needs to be https://github.com/cantaloupe-project/cantaloupe/releases/download/v5.0.4/cantaloupe-5.0.4.zip so we go for a fixed version, but as said in a previous comment we should go for
CANTALOUPE_VERSION in every mention of the mention in the following commands. That way in a future pull we only change that.